### PR TITLE
Portfolio2

### PR DESCRIPTION
**Pull request by Melaku05**
**. Portfolio project pull request**
-  added contact form in the footer
- form with validation  input
- all fields are required

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -371,20 +371,23 @@ <h2 class="form-text">
         chat please get in touch.
       </h2>
       <div class="form">
-        <form>
+        <form action="https://formspree.io/f/mzborojb" method="POST">
```
Comment (medaminedev66) [inline index.html:374]: - [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided.
- [ ] I would highly recommend using the same fonts that are provided in the Figma template so your UI design is professional.
- Your design:
image

- Figma design:
image

File: index.html [hunk 2]
```diff
@@ -393,6 +396,8 @@ <h2 class="form-text">
             id="feedback"
             name="feedback"
             placeholder="Enter text here"
+            required 
+            maxlength="500"
           ></textarea>
           <div class="btn-align">
             <input type="submit" class="btn-3" value="Get in Touch" />
```
Comment (medaminedev66) [inline index.html:403]: - [ ] the social icon image is broken kindly fix it.
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Melaku05 and @ekenecf , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️ The form is submitted successfully with valid input values. Great job 👍

## Required Changes ♻️

- [x] Pull Request should have a descriptive description telling others what changes are inside so they can read what is being worked on. Kindly change it 😃

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Melaku05) [reply index.html:374]: Thank you, I just modify the style of form

**Review by keroloussamy — APPROVED** [review]
Hi @Melaku05, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**